### DISCUSSION TITLE
Scientific Notation?

### DISCUSSION THREAD

**Discussion by Izzy129** [johannes-wolf/rpnspire#39] [Q&A] [answered]
Is it possible to output results in scientific notation? (i.e. 100 being output as 1.00x10^2)
If so, how can I do so, thanks in advance.

**Comment by johannes-wolf** [comment]
Hi, I am currently doing a rewrite of the whole application (see branch `refactor/ui-lib`) I will take this into account. The current version has no support for setting the output format. If you have feature requests, do not hesitate opening an issue :).

**Comment by johannes-wolf** [comment, marked as answer]
I have pushed a new version (+ release) on master that should fix this issue.

**Reply by Izzy129** [reply]
Thank you! Best of luck on the rewrite👍

**Reply by johannes-wolf** [reply]
Thanks. I discovered that my “fix” disables assignments. Hope this is not a problem. Will maybe apply a real fix. But I will also release the first “new” version of RPNspire soon, which has a working solution.
